client-app/core/api/graphql/whiteLabeling/queries/index.ts

```diff
@@ -1,12 +1,54 @@
 import { useLazyQuery } from "@vue/apollo-composable";
-import { GetWhiteLabelingSettingsDocument } from "@/core/api/graphql/types";
```

Comment:
keep it and try @recursive
https://docs.developers.optimizely.com/platform-optimizely/v1.4.0-optimizely-graph/docs/recursive-directive-usage#:~:text=ContentLink%20%40recursive(depth%3A%203)

Reply:
Let's keep the manual recursive methods for now since `@recursive` directive is not supported.